grommet/grommet
# Updating Select theme doc.js to correctly reference `select.options.box` instead of `select.options.container`

Signed-off-by: Matt Glissmann <mdglissmann@gmail.com>



#### What does this PR do?
Updates `Select/doc.js` to reference `select.options.box' instead of `select.options.container` which does not exist in `themes/base.js`.

#### Where should the reviewer start?
`Select/doc.js`

#### What testing has been done on this PR?
- manual testing in local project
- added tests for:
  - apply new documentation definition: select.options.container
  - confirm backwards compatibility support for legacy select.options.box
  - apply both select.options.container and select.options.box styling when both are present; select.options.container is prioritized and applied if conflict between the two.

#### How should this be manually tested?

#### Any background context you want to provide?
https://github.com/grommet/grommet/issues/3183

#### What are the relevant issues?
- https://github.com/grommet/grommet/issues/3183
- https://github.com/grommet/grommet/pull/3184#issuecomment-508264393

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
Yes.

#### Should this PR be mentioned in the release notes?
No.

#### Is this change backwards compatible or is it a breaking change?
Backwards compatible.
